tests/integration/test_route53resolver.py

```diff
@@ -0,0 +1,560 @@
+                ResolverRuleId=resolver_rule_id
+            )
+            snapshot.match(f"delete_resolver_rule_res{ind}", delete_resolver_rule)
+
+    @pytest.mark.aws_validated
+    def test_delete_non_existent_resolver_rule(self, route53resolver_client, snapshot):
+        resolver_rule_id = "id-123"
+        with pytest.raises(
+            route53resolver_client.exceptions.ResourceNotFoundException
+        ) as resource_not_found:
+            route53resolver_client.delete_resolver_rule(ResolverRuleId=resolver_rule_id)
+        snapshot.match("resource_not_found_res", resource_not_found.value.response)
+
+    @pytest.mark.aws_validated
+    def test_disassociate_non_existent_association(self, route53resolver_client, snapshot):
+        with pytest.raises(
+            route53resolver_client.exceptions.ResourceNotFoundException
+        ) as resource_not_found:
+            route53resolver_client.disassociate_resolver_rule(
+                ResolverRuleId="rslvr-123", VPCId="vpc-123"
+            )
+        snapshot.match("resource_not_found_res", resource_not_found)
+
+    @pytest.mark.aws_validated
+    def test_create_resolver_query_log_config(self, route53resolver_client, cleanups, snapshot):
+        snapshot.add_transformer(snapshot.transform.key_value("Name", "name"))
+        request_id = short_uid()
+        result = route53resolver_client.create_resolver_query_log_config(
+            Name=f"test-{short_uid()}",
+            DestinationArn="arn:aws:logs:us-east-1:123456789012:log-group:sampletest123",
+            CreatorRequestId=request_id,
+        )
+        create_rqlc = result.get("ResolverQueryLogConfig")
+        resolver_config_id = create_rqlc["Id"]
+        if self._wait_created_log_config_is_listed_with_status(
+            route53resolver_client, resolver_config_id, "CREATED"
+        ):
+            create_rqlc["Status"] = "CREATED"
+
+        snapshot.match("create_resolver_query_log_config_res", create_rqlc)
+
+        delete_resolver_config = route53resolver_client.delete_resolver_query_log_config(
+            ResolverQueryLogConfigId=resolver_config_id
+        )
+        snapshot.match("delete_resolver_query_log_config_res", delete_resolver_config)
+
+    @pytest.mark.aws_validated
+    def test_delete_non_existent_resolver_query_log_config(self, route53resolver_client, snapshot):
+        resolver_rqlc_id = "test_123"
+        with pytest.raises(
+            route53resolver_client.exceptions.ResourceNotFoundException
+        ) as resource_not_found:
+            route53resolver_client.delete_resolver_query_log_config(
+                ResolverQueryLogConfigId=resolver_rqlc_id
+            )
+        msg = resource_not_found.value.response["Message"].split(".")[0]
+        snapshot.add_transformer(snapshot.transform.jsonpath("$..Error.Message", msg))
+        snapshot.add_transformer(snapshot.transform.jsonpath("$..Message", msg))
```

Comment:
```suggestion
```

Reply:
@dominikschubert at these places the error msg is partially similar in aws and localstack, so tried to add a meaning msg rather than skipping it. Had fixed at most of the places by patching in provider.py.

Reply:
Transformers should generally only be used to make snapshots compatible between different runs against AWS. If a bug/missing feature in localstack prevents them from passing the corresponding parts should be skipped from verification by using the `skip_snapshot_verify` pytest marker.

If you just need to patch the resource name in the error message it would make sense to just do just that with something like: `snapshot.add_transformer(snapshot.transform.regex("whatevervalueyouwanttoreplace", "<token>"))`

Reply:
Right, so i will rather skip checking the err message in this case rather than partially asserting it. Thank you.